apps/sms/style/sms.css

```diff
@@ -933,16 +932,29 @@ body[role="application"] form[role="dialog"][data-type="action"] > header:first-
 }
 
 /* Phone details label styles */
-.phone-type,
+.carrier-information {
+  display: flex;
+  flex-flow: row nowrap; 
+}
+
+.phone-type, .phone-carrier {
+  min-width: 0px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  flex: 0 1 auto;
+}
+
+.phone-number {
+  flex: 1 0 auto;
```

Comment:
as I told you on IRC, you need "flex: none" so that it looks like current master